components/resources/ResourceOccurrences.js

```diff
@@ -23,11 +23,9 @@ import { useFetch } from "hooks/useFetch";
 const ResourceOccurrences = (props) => {
   const { resourceUri } = props;
 
-  const { data, loading } = useFetch(
-    resourceUri
-      ? `/api/occurrences?resourceUri=${encodeURIComponent(resourceUri)}`
-      : null
-  );
+  const { data, loading } = useFetch(resourceUri ? `/api/occurrences` : null, {
```

Comment:
Missed this the first time, but isn't this ternary handled by `useFetch` now? 

https://github.com/rode/rode-ui/blob/f5c1104e87d9cea414866af8a2b2dc340e70dc98/hooks/useFetch.js#L26